beetmoverscript/utils.py

```diff
@@ -65,22 +65,28 @@ def generate_beetmover_template_args(task, release_props):
         "platform": release_props["platform"],
     }
 
+    if cert != 'nightly':
+        template_args["build_number"] = release_props["buildnumber"]
```

Comment:
We need to find a way to pass it.  The easiest way might be the task definition, since the decision task knows the build number, and it modifies the beetmover task definition.  If we want it in balrog_props, we need to pass it into the build, and have the build put it into balrog_props, and then beetmover reads it from balrog_props, which is an extra step worth of work.